### DISCUSSION TITLE
Kami Mini keeps restarting

### DISCUSSION THREAD

**Discussion by PTechHD** [roleoroleo/yi-hack-Allwinner-v2#273] [Q&A]
I was so happy to have found this firmware for my Kami. I checked the SN and firmware and my camera is a match for being able to use the firmware. The instructions on the main page say to simply unzip the contents onto the root of a memory card that was formatted in the camera. I've done this with memory cards from 64gb to 256gb in volume and when I insert the memory card, the camera simply keeps restarting. It doesn't seem to install anything and when I navigate to the page that should show the settings the page doesn't load. What am I doing wrong? The instructions are simple enough, yet it doesn't seem to work.

**Comment by roleoroleo** [comment]
Try a smaller sd card.

**Reply by PTechHD** [reply]
ive tried 64gb 128gb 256gb. I read that 64gb would be fine. how small does the card need to be?

**Reply by roleoroleo** [reply]
I don't know. It depends on partition, allocation unit size, etc...
Contrary to what my wife says, smaller is better. :)

**Reply by ghost** [reply]
Maybe you can partition 64GB to <32GB?

**Reply by PTechHD** [reply]
Just purchased 3 micro sd cards for my cameras. Im not in the US so getting this stuff takes time. I purchased 32gb memory cards and the software still does not install. Camera boots up and they can even do the firmware backups successfully, but they will not install this hacked firmware. Im not certain, but I see no noticable differences and the website http://ip-cam:8080/ does not load, so I'm just assuming that this is not working

**Reply by roleoroleo** [reply]
Try to restart from scratch:

1. Sync your cam with the app and check if it works correctly.
2. Clean the sd card.
3. Apply the hack.
4. Get the ip of the cam in your wireless network (check the home networking page of your router).
5. Try to connect to the web page http://IP_ABOVE:8080
